SQLQuery4.sql

```diff
@@ -0,0 +1,6 @@
+update tblEpisode
+set Title=concat (Title ,'_canseld')
+where EpisodeId in
+(select EpisodeId
+from tblEpisode
+where DoctorId is null)
```

Comment:
de we really need to do a nested query to check if the episode has zero doctor appearance?

Reply:
No, it's my fault.
I'll adjust that.